### DISCUSSION TITLE
How to turn off / disable Samba Service

### DISCUSSION THREAD

**Discussion by oxygi** [MiczFlor/RPi-Jukebox-RFID#2244] [Q&A] [answered]
Hi, I will not use the Windows shared folder. I am using WINSCP or Webapp. Is it possible to deactivate or/and remove samba?
I am on Version [v2.5.0]

**Comment by s-martin** [comment, marked as answer]
You can disable the samba service, see here: https://forums.raspberrypi.com/viewtopic.php?t=264140

**Comment by AlvinSchiller** [comment]
You can also try out the newer [version3](https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/future3/main/documentation/builders/installation.md), if it contains [all features you need](https://github.com/MiczFlor/RPi-Jukebox-RFID/blob/future3/main/documentation/developers/status.md).
It has a setup step for the de-/activation of samba.
